modules/tinymce/src/themes/silver/test/ts/browser/editor/contexttoolbar/ContextToolbarIframePositionTest.ts

```diff
@@ -37,7 +39,7 @@ describe('browser.tinymce.themes.silver.editor.contexttoolbar.ContextToolbarIFra
     toolbar: 'fullscreen',
     height: 400,
     base_url: '/project/tinymce/js/tinymce',
-    content_style: 'body, p { margin: 0; } tr { height: 36px; }',
+    content_style: 'body, p { margin: 0; line-height: 22px; font-family: sans-serif; } tr { height: 36px; }',
```

Comment:
Why was the font-family changed?

Reply:
I had mentioned this in stand up, but it's because the default font family tries to use "System default" which varies between browsers. As such, it causes the selection heights to be different and causes differing behaviour so we need to make sure we're using the same font on all OSs.